src/core/connection.c

```diff
@@ -6612,6 +6612,7 @@ QuicConnGetV2Statistics(
     Stats->RecvDecryptionFailures = Connection->Stats.Recv.DecryptionFailures;
     Stats->RecvValidAckFrames = Connection->Stats.Recv.ValidAckFrames;
     Stats->KeyUpdateCount = Connection->Stats.Misc.KeyUpdateCount;
+    Stats->ChangeSrcCidCount = Connection->Stats.Misc.ChangeSrcCidCount;
```

Comment:
Look at the comment below about needing to be size checked. Please move this down after `SendCongestionWindow`.